src/UI/Components/InlineEditable/EditableSelectField.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { Maybe } from "@/Core";
+import { CreatableSelectInput } from "@/UI/Components/CreatableSelectInput";
+import { EditableField, FieldProps } from "./EditableField";
+
+interface Props extends FieldProps {
+  options: string[];
+  onCreate: (value: string) => Promise<Maybe.Type<string>>;
+}
+
+export const EditableSelectField: React.FC<Props> = ({
```

Comment:
I think that since this is added, the `InlineSelectOption` component can be removed